mender.go

```diff
@@ -213,7 +213,7 @@ func getManifestData(dataType, manifestFile string) string {
 	// This is where Yocto stores buid information
 	manifest, err := os.Open(manifestFile)
 	if err != nil {
-		log.Errorf("Can not read manifest field '%s' from file: %s", dataType, manifestFile)
+		log.Errorf("device_type file not found. Unable to verify the existing hardware. Update will continue anyways, '%s' from file: %s", dataType, manifestFile)
```

Comment:
This section is shared and called by several functions with different files. I suggest you put the check and the message in the function that calls `GetDeviceType()` instead.

Also, consider including the `err.Error()` string somewhere in the log output, because you don't actually know that the file cannot be found, only that it cannot be opened, which could be because permissions are wrong. Standard practice in Mender is to do something like: `log.Errorf('%s' could not be opened: %s", file, err.Error())` (colon separating high level error from low level error).